### PR TITLE
Proposed update to Figure 5 in paper 

### PR DESCRIPTION
**Pull request by kponder**
Alex and I talked about ways to make Figure 5 more clear and here is a suggestion. 

One of the main questions is, do you need to boxes illustrating the decrease in scale in each plot?

### CODE REVIEW

**Comment by juramaga** [comment]
I guess I don't really see a change in the plot apart from rearranging the boxes and changing colors. It's a complex plot for the non-expert, I had some time grasping it. What is the point we want to make here? It does not come obvious to me from the plot that brier is more sensitive that logloss to systematics.

**Comment by kponder** [comment]
Hi @juramaga , basically the point is the superficial changes. I didn't want to change the *content* just how it was presented. I changed 3 main things:

1. **Removed the insets**: The original right two plots were coded as zoom-ins of the plot to its left, which resulted in extra black lines on the plots. These lines crossed through data points. My group read this paper for journal club and we all initially thought the line in the first plot on the left was some kind of fit. This is not the case and so I felt it was a detriment to the plot. 
    * I left some form of the boxes to illustrate the scale differences, but this goes to my original question of "are these boxes really necessary"?
2. **Making the rightmost plot larger**: The plot furthest to the right on the original version (non-subsumed) has a lot of overlapping data points. I understand that some of them are supposed to overlap, but it was hard to distinguish the different shapes. By making this plot much larger, it is easier to distinguish the different scenarios. The other two plots only have a single line of points and so they were not as crowded and could be made smaller. 
3. **The color scheme**: I felt like it was hard to distinguish some of the blues and greens that overlapped. Some of the greens rendered fairly faintly. 

In summary, the point of these changes is to improve presentation and not content so that the reader (or person in the audience of a presentation) can more easily understand the takeaways from the plot.

I should have specified these changes and why I made them in my original post instead of being so vague!

**Comment by aimalz** [comment]
Thanks @kponder!  I think the new plot achieves all its goals and look forward to propagating the new color scheme throughout the other plots, assuming the aesthetics are acceptable to our resident experts @reneehlozek and @michellelochner.  I'll leave this PR open until the referee comments come in, in case they have additional feedback on the plots.

**Comment by reneehlozek** [comment]
I think these colours are good, and like the new figure/inset, although @tinapeters had raised a comment that the inset boxes are a little odd in the points in the zoomed plots are not the same as those in the inset (grey) boxes... so we should be careful in explaining this. While we are on the topic  I also have some thoughts about the discussion of this plot in the text. If I'm free to edit let me know and I can make a PR.

**Review by aimalz — CHANGES_REQUESTED** [review]
Great work @kponder!  Let's propagate the color changes through the notebook before merging.

**Review by aimalz — APPROVED** [review]
